.conan/test_package/conanfile.py

```diff
@@ -3,19 +3,27 @@
 from conan import ConanFile
 from conan.tools.cmake import CMake, cmake_layout
 from conan.tools.build import can_run
+from conan.tools.files import save, load
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "CMakeToolchain", "CMakeDeps"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
+    test_type = "explicit"
```

Comment:
For Conan 1.x only: https://docs.conan.io/1/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe